docs/RFCS/20220608_split_jobs_system_tables.md

```diff
@@ -0,0 +1,279 @@
+
+---
+
+### system.job_status_progress + system.job_status_message
+
+#### Description
+
+The job_status_progress and system.job_status_messages tables store human-readable information about a job's current and recent status and progress.
+
+These tables are not read by the jobs control system at any point, and should not be read by the job itself either; they are intended solely for human-consumption via SHOW JOBS and the DB Console. 
+
+Both tables have many rows per job: Each time a job updates its persisted status or progress it will insert a new row into the respective table with the new value, and inspection tools will scan for the latest row per job to find its current status/progress. 
+
+Both tables have the same primary key: job ID plus row creation time. However they are separate as their payloads are both of very different size -- a small number versus a large string -- and are updated at very different frequencies; a completion percentage or timestamp likely updates continuously for the entirely life of a job while execution status may change a fixed number of times per execution phase.
+
+#### Schema
+
+```
+    CREATE TABLE system.job_status_progress (
+    	job_id INT NOT NULL,
+    last_updated TIMESTAMPTZ,
+     	percent_complete FLOAT, # null if job does not have a defined completion percentage
+    	timestamp_complete DECIMAL, # set only for jobs that advance in time e.g. cdc or replication
+    	PRIMARY KEY (job_id, last_updated DESC)
+    )
+    CREATE TABLE system.job_status_message (
+    	job_id INT NOT NULL,
+    last_updated TIMESTAMPTZ,
+    	status STRING, # like v1's running_status, e.g. "waiting for leases" or "importing rows"
+    	detail STRING, # more info, reason, etc.
+    	PRIMARY KEY (job_id, last_updated DESC)
+    )
+```
+
+
+
+#### SHOW JOBS
+
+The job control table, along with these tables plus the intake table, defines the set of jobs the system is running, will run or has has failed or creased running, along with human-readable information about them, such as their description, percentage complete, or  recent status messages; taken together, these what an operator wants to know when they ask "what jobs is my cluster running? How complete are the job I was running? Did they fail and if so why?"  This interaction is fulfilled by SHOW JOBS and the jobs page on the DB console. Both of these are powered by an internal virtual table crdb_internal.jobs, and will continue to be going forward, however this table will now be reading columns only from the control table and the status tables (plus the intake table). Rendering the virtual table will _not_ require decoding large proto payloads, that also contain large amounts of jon-specific state, nor will it block on transactions that are manipulating that state, as the virtual table will not depend on the info table, described below, where that state now resides and is manipulated.
```

Comment:
Nit: `s/creased/ceased/g`

PS It is easier to comment on specific lines in a paragraph if you flow the paragraph in your editor instead of using one line per paragraph. Markdown works the same regardless.